.licenserc.json

```diff
@@ -94,5 +94,37 @@
     " * See the License for the specific language governing permissions and",
     " * limitations under the License.",
     " */"
+  ],
+  "**/*.ts": [
+    "/* Copyright 2020 Energinet DataHub A/S",
+    " *",
+    " * Licensed under the Apache License, Version 2.0 (the \"License2\");",
+    " * you may not use this file except in compliance with the License.",
+    " * You may obtain a copy of the License at",
+    " *",
+    " *     http://www.apache.org/licenses/LICENSE-2.0",
+    " *",
+    " * Unless required by applicable law or agreed to in writing, software",
+    " * distributed under the License is distributed on an \"AS IS\" BASIS,",
+    " * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+    " * See the License for the specific language governing permissions and",
+    " * limitations under the License.",
+    " */"
+  ],
+  "**/*.scss": [
+    "/* Copyright 2020 Energinet DataHub A/S",
+    " *",
+    " * Licensed under the Apache License, Version 2.0 (the \"License2\");",
+    " * you may not use this file except in compliance with the License.",
+    " * You may obtain a copy of the License at",
+    " *",
+    " *     http://www.apache.org/licenses/LICENSE-2.0",
+    " *",
+    " * Unless required by applicable law or agreed to in writing, software",
+    " * distributed under the License is distributed on an \"AS IS\" BASIS,",
+    " * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+    " * See the License for the specific language governing permissions and",
+    " * limitations under the License.",
+    " */"
   ]
```

Comment:
suggestion: add an `"**/*.html"` entry using an HTML comment block (`<!-- <license --->`) same as the `"**/*.csproj"` entry

suggestion: add `"**/*.js"` and `"**/*.mjs"` entries using the same comment as the `"**/*.ts"` entry